lock-api/src/main/java/com/palantir/lock/client/LockTokenShare.java

```diff
@@ -24,7 +24,7 @@
 import com.palantir.lock.v2.LockToken;
 import com.palantir.logsafe.Preconditions;
 
-final class LockTokenShare implements LockToken {
+public final class LockTokenShare implements LockToken {
```

Comment:
At this point maybe we should just add something to the LockToken interface that is going to return a String that can be used for logging? Or make it extend https://github.com/palantir/safe-logging/blob/develop/safe-logging/src/main/java/com/palantir/logsafe/SafeLoggable.java?